app/controllers/messages_controller.rb

```diff
@@ -6,10 +6,6 @@ class MessagesController < ApplicationController
     controller.ensure_logged_in t("layouts.notifications.you_must_log_in_to_send_a_message")
   end
 
-  before_filter do |controller|
-    controller.ensure_authorized t("layouts.notifications.you_are_not_authorized_to_do_this")
-  end
-
```

Comment:
When this filter is now dropped from all these controllers what is now taking care of this action (ensuring only authorized person can view the content and show error message in case of unauthorized access)?